src/App.js

```diff
@@ -1,28 +1,18 @@
-import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import MyNavbar from './components/Navbar';
+import Locations from './pages/locations';
+import CountryDetails from './pages/details';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <MyNavbar />
+      <Routes>
+        <Route exact path="/" element={<Locations />} />
+        <Route exact path="/details" element={<CountryDetails />} />
```

Comment:
- [ ] For showing the specific countries list the route URL depends on the `country name` or some unique key.For ex:-
```
<Route exact path="/item/:id" element={Item} />
```
Here `id` is a unique key for specific items, same you can show the `country name` or `country_id` as a unique key.

![Screenshot from 2023-01-12 22-04-49](https://user-images.githubusercontent.com/26767383/212127237-32da002a-e7fb-426d-b02a-c14540c7475a.png)